File: android/app/src/main/java/com/faceproject/android/model/UserProfile.kt
package com.faceproject.android.model

import com.google.gson.annotations.SerializedName

data class UserProfile(
    @SerializedName("name") val name: String,
    @SerializedName("age") val age: Int,
    @SerializedName("nickname") val nickname: String?,
    @SerializedName("gender") val gender: String,
    @SerializedName("height_cm") val heightCm: Double,
    @SerializedName("weight_kg") val weightKg: Double?,
    @SerializedName("registered_at") val registeredAt: String
)

File: ios/FaceAuth/Models/UserProfile.swift
import Foundation

struct UserProfile: Codable, Equatable {
    let name: String
    let age: Int
    let nickname: String?
    let gender: String
    let heightCm: Double
    let weightKg: Double?
    let registeredAt: String

    enum CodingKeys: String, CodingKey {
        case name, age, nickname, gender
        case heightCm = "height_cm"
        case weightKg = "weight_kg"
        case registeredAt = "registered_at"
    }
}
